test/Api.Tests.Nunit/TestStartup.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Api.Managers;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using MyTested.AspNetCore.Mvc;
+using Api.Permission;
+
+namespace Api.Tests.Nunit
+{
+    public class TestStartup : Startup
+    {
+        public TestStartup(IHostingEnvironment env) : base(env) { }
+
+        public void ConfigureTestServices(IServiceCollection services)
+        {
+            base.ConfigureServices(services);
```

Comment:
Simply inherit 'Startup' will not work, because you to exchange the postgreSQL connector. Otherwise you have to configure one.